applications/HDF5Application/python_scripts/single_mesh_temporal_input_process.py

```diff
@@ -9,7 +9,9 @@ def Factory(settings, Model):
     default_settings = KratosMultiphysics.Parameters("""
             {
                 "model_part_name" : "MainModelPart",
-                "file_settings" : {},
+                "file_settings" : {
+                    "file_access_mode" : "read_only"
```

Comment:
Are we sure that we want "read_only" to be the default setting? I think some truncate+write mode would make more sense as a default, since input processes are relatively few, but your call.

Reply:
After looking into it, both single_mesh_temporal_input_process.py and single_mesh_initialization_process.py set the file access mode to read_only to override the "exclusive" default setting of the SerialFileFactory. Either process would fail if opening in exclusive mode (it throws an error if the file exists), so I'm afraid I need to set them both. Note that this is a modification of an existing process, which I now realize I should have pushed as a separate PR.

Reply:
I don't see `read_only` in any python script in the hdf5 app in master branch. Let's discuss it in person.